main/main.cpp

```diff
@@ -1146,8 +1146,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 #ifdef TOOLS_ENABLED
 		editor = false;
 #else
-		const String error_msg = "Error: Couldn't load project data at path \"" + project_path + "\". Is the .pck file missing?\nIf you've renamed the executable, the associated .pck file should also be renamed to match the executable's name (without the extension).\n";
-		OS::get_singleton()->print("%s", error_msg.utf8().get_data());
+		const String error_msg = "Error: Couldn't load project data at path \"" + project_path + "\". Is the .pck/.zip file missing?\nIf you've renamed the executable, the associated .pck/.zip file should also be renamed to match the executable's name (without the extension).\n";
+		OS::get_singleton()->print("%s", error_msg.ascii().get_data());
```

Comment:
Why? `project_path` can contain Unicode.

Reply:
Sorry, that was a rebasing mistake.